### ISSUE TITLE
Typedoc validator should verify if an item exists in a parent class

### ISSUE THREAD

**Issue by pomek** [ckeditor/ckeditor5#13298]
### Scenario from CKEditor 5 code base

* `engine/model/range` defines a method that has a link in its description that points to the `Range` class https://github.com/ckeditor/ckeditor5/blob/c8a89f383efcd7d6369012039b2db6409d419467/packages/ckeditor5-engine/src/model/range.ts#L166-L192
* `engine/model/liverange` extends the `Range` class https://github.com/ckeditor/ckeditor5/blob/c8a89f383efcd7d6369012039b2db6409d419467/packages/ckeditor5-engine/src/model/liverange.ts#L31
* The `~Range` link is invalid from the `module:engine/model/liverange~LiveRange` class, as the `liverange` module does not contain the `Range` class.
* The linker mechanism should know the context for resolving links. If a link is unavailable in the current processed class, perhaps we should check if it exists in a parent class.
* A place to update: https://github.com/ckeditor/ckeditor5-dev/blob/a3bb64b0120c7c166c1edf04e578ce47e58b0320/packages/ckeditor5-dev-docs/lib/validators/link-validator/index.js#L19.

**Comment by pomek** [comment]
@psmyrek, didn't we agree to skip verifying inherited properties?

**Comment by psmyrek** [comment]
Yes. It seems that skipping the check for inherited properties is correct, as they will be checked in their base class/interface anyway.
